### PR TITLE
Adding pytest

### PR DESCRIPTION
**Pull request by MatanP12**
- Adding pytest as dev dependency
- creating a dummy test that always pass
- Adding pytest as github action.

fix : #17

### CODE REVIEW

**Review by TomerNewmanPrograms — DISMISSED** [review]
LGTM

**Review by OmerCS8 — DISMISSED** [review]
LGTM

**Review by AlonShakaroff — CHANGES_REQUESTED** [review]
Lol, why not just assert == true?
Other than that looks good to me

**Comment by MatanP12** [comment]
> Lol, why not just assert == true? Other than that looks good to me

As long it asserts some true value and it pass the tests, I don't think it matters :P

File: Pipfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ django = "*"
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
```
Comment (natifridman) [inline Pipfile:11]: Why do you need the `pytest` package? Isn't `pytest-django` enough?

**Review by natifridman — COMMENTED** [review]
We usually make the first test fail so we know that the CI works as expected

File: Pipfile (hunk 1 above)
Comment (MatanP12) [reply Pipfile:11]: I followed Edward installing guide, from my understanding, `pytest-django` is a plugin for `pytest`.

**Comment by natifridman** [comment]
@MatanP12 Now that we know the tests work, you can fix it so it won't block us

**Comment by natifridman** [comment]
> So for future PRs, we should add a test that always fails?

No need, I meant for the first run of the action when you create it.

**Comment by MatanP12** [comment]
@natifridman 

Can you re run the CI?

**Comment by abiton1** [comment]
LGTM

**Review by abiton1 — APPROVED** [review]

**Review by natifridman — APPROVED** [review]

**Review by yulisuliman — APPROVED** [review]
Looks good

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]
LGTM

**Pull request merged**